modin/pandas/test/test_groupby.py

```diff
@@ -461,7 +477,7 @@ def test_aggregate_alias():
 @pytest.mark.parametrize(
     "col1_category", [True, False], ids=lambda v: f"col1_category={v}"
 )
-def test_simple_row_groupby(by, as_index, col1_category):
+def test_simple_row_groupby(by, as_index, col1_category, request):
```

Comment:
Why is request added? Do we use it somewhere?

Reply:
Removed